docu/sphinx/source/_static/custom.css

```diff
@@ -0,0 +1,3 @@
+dl.function {
```

Comment:
Doing this in this way you have the first entry flushed to the headline and the last entry will leave a big gap to the footer. There are two solutions:

1. Split the margin into `margin-top` and `margin-bottom`.
2. Use `dl.function:not(:last-child)` as the selector.

I recommend the second one.

Reply:
Thanks for the review. I've applied all suggestions! See https://docs-staging.byte-physics.de/igor-unit-testing-framework which I deployed by executing the deployment-staging job.

Reply:
Building this locally on my computer produces the correct page:

![grafik](https://user-images.githubusercontent.com/12125053/211291544-c556b890-d320-40d8-95fb-d203a0b229d0.png)

But when I look at the staging branch using your link I will still see the old code:

![grafik](https://user-images.githubusercontent.com/12125053/211291716-0c741b8a-485c-4af2-b6a7-7d64f4083dae.png)

Reply:
The artifacts from `documentation` step are correct. The error must be in `deployment staging`.

Reply:
My Browser Cache tricked me. It's working now. We can use version specific links for assets to prevent this in the future but I don't know if we change the assets that often.

Reply:
> We can use version specific links for assets to prevent this in the future but I don't know if we change the assets that often.

Normally we don't chang them often.